--- os_base.features.wysiwyg.inc.old	2012-09-10 08:00:52.920891863 -0700
+++ os_base.features.wysiwyg.inc	2012-09-10 08:01:36.369107319 -0700
@@ -64,13 +64,40 @@
           'italic' => 1,
           'underline' => 1,
           'strikethrough' => 1,
+          'justifycenter' => 1,
+          'justifyright' => 1,
           'bullist' => 1,
           'numlist' => 1,
+          'outdent' => 1,
+          'indent' => 1,
+          'undo' => 1,
+          'redo' => 1,
           'link' => 1,
+          'image' => 1,
           'forecolor' => 1,
           'backcolor' => 1,
+          'copy' => 1,
+          'paste' => 1,
           'removeformat' => 1,
         ),
+        'advimage' => array(
+          'advimage' => 1,
+        ),
+        'font' => array(
+          'fontsizeselect' => 1,
+        ),
+        'paste' => array(
+          'pasteword' => 1,
+        ),
+        'style' => array(
+          'styleprops' => 1,
+        ),
+        'table' => array(
+          'tablecontrols' => 1,
+        ),
+        'media' => array(
+          'media' => 1,
+        ),
         'drupal' => array(
           'media' => 1,
         ),
@@ -87,6 +114,7 @@
       'paste_auto_cleanup_on_paste' => 0,
       'block_formats' => 'p,address,pre,h2,h3,h4,h5,h6,div',
       'css_setting' => 'theme',
+      'css_theme' => 'seven',
       'css_path' => '',
       'css_classes' => '',
     ),
